packages/core/src/components/sections/ProductGallery/EmptyGallery.tsx

```diff
@@ -1,37 +1,44 @@
-import { Icon, LinkButton } from '@faststore/ui'
-
-import EmptyState from 'src/components/ui/EmptyState'
+import {
+  Icon as UIIcon,
+  LinkButton as UILinkButton,
+  EmptyState as UIEmptyState,
+} from '@faststore/ui'
 
 function EmptyGallery() {
   return (
-    <EmptyState variant="rounded">
-      <header data-fs-empty-state-title>
-        <Icon name="CircleWavyWarning" width={56} height={56} weight="thin" />
-
-        <p>Nothing matches with your search</p>
-      </header>
-
-      <LinkButton
-        data-fs-empty-state-link
+    <UIEmptyState
```

Comment:
Can we fix this extra space between the `breadcrumb` and the `empty state`?

<img width="1359" alt="image" src="https://user-images.githubusercontent.com/11613011/232591751-4c3197e7-0fda-4125-9b2c-62ff0dec87f6.png">

Reply:
Fixed [here](https://github.com/vtex/faststore/pull/1716/commits/907bfdef05db595b3954dbe95b8999226ac2fece)

<img width="1520" alt="image" src="https://user-images.githubusercontent.com/3356699/232625237-85ad60ae-65e9-489e-a57d-668104b601f3.png">